### PR TITLE
Investigator (Update)

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Removed bar and microwave.
- Removed salvage equipment.
- Created separate engine room.
- Reworked atmos.
- Added guidebook entry with map.

## Why / Balance
Part of the ongoing push for limiting shuttles to single purpose. Investigator enjoyers had it too good for far too long.

## How to test
Buy NR Investigator and despair.

## Media

image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Investigator powergaming

**Changelog**
:cl: erhardsteinhauer
- tweak: Updated NR Investigator.

### CODE REVIEW

**Comment by ErhardSteinhauer** [comment]
Ship owner: @UbaserB 
(Just tagging you to make sure you're aware there's suggested changes for you ship.)

**Comment by dvir001** [comment]
Might want to keep microwave just for the rads reactions

**Comment by ErhardSteinhauer** [comment]
> Might want to keep microwave just for the rads reactions

They can research service branch and print microwave.
Or buy refined bananium.

**Comment by ErhardSteinhauer** [comment]
Or spray it

**Comment by dvir001** [comment]
> > Might want to keep microwave just for the rads reactions
> 
> They can research service branch and print microwave. Or buy refined bananium.

Rads removed from it upstream and we probably just going to keep that change

**Comment by ErhardSteinhauer** [comment]
> > > Might want to keep microwave just for the rads reactions
> > 
> > 
> > They can research service branch and print microwave. Or buy refined bananium.
> 
> Rads removed from it upstream and we probably just going to keep that change

Hm, riiiight. Still, research option is there.

**Comment by UbaserB** [comment]
I don't like the tabletop computers, i prefer normal ones. Also, there's a shutter over the door in the control room. Other than that, lgtm

**Comment by ErhardSteinhauer** [comment]
> I don't like the tabletop computers, i prefer normal ones. Also, there's a shutter over the door in the control room. Other than that, lgtm

Shutter over the door is intentional

**Comment by dvir001** [comment]
> > > > Might want to keep microwave just for the rads reactions
> > > 
> > > 
> > > They can research service branch and print microwave. Or buy refined bananium.
> > 
> > 
> > Rads removed from it upstream and we probably just going to keep that change
> 
> Hm, riiiight. Still, research option is there.

Fair

**Comment by UbaserB** [comment]
> > I don't like the tabletop computers, i prefer normal ones. Also, there's a shutter over the door in the control room. Other than that, lgtm
> 
> Shutter over the door is intentional

I can see the idea you're going for, but generally please just keep the shutters external (so like, between the seating area and the control room, no shutters)

**Comment by ErhardSteinhauer** [comment]
> but generally please just keep the shutters external

I disagree with this.
However, I don't have too strong feelings about it and I have editor opened right now, so I'll remove it.

**Comment by ThatOneGoblin25** [comment]
Hi do you reckon you could semi-revert the artifact chamber from the old version to have a gap between the ship and the chamber?

20240326211859!Investigator

I recently got told that the firelocks can be a bit of a death trap and I only made that change because the artifact analyse console had a smaller range.

**Comment by ErhardSteinhauer** [comment]
I love me a good death trap. And I like the firelock setup there. But I'll think about it more.

File: Resources/ServerInfo/_NF/Guidebook/Shipyard/Investigator.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  
+  - Check that the gyroscope is anchored, powered, and enabled.
+  - Check that the mini gravity generator is anchored, powered, and enabled.
+  - Check that the blast doors are closed.
+  
+  ## Sidenotes
+  
+  * - Investigator-class research ships are equipped with two APC units that can be used to appraise the ship's total power consumption. Unmodifued Lantern-class ship requires 39.6 kW of power to remain operational.
+  
+  ** - Investigator-class research ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, however the ship comes equipped with additional P.A.C.M.A.N. generator unit and 67 solar panels that can provide up to 33.5 kW of power under the ideal circumstances: assuming that on average properly maintained solar array will provide one third of maximum output power, recommended target power output for each P.A.C.M.A.N. generator units is 15 kW.
```
Comment (whatston3) [inline Resources/ServerInfo/_NF/Guidebook/Shipyard/Investigator.xml:105]: Small suggested rewrite.
```suggestion
  * - Investigator-class research ships are equipped with two APC units that can be used to appraise the ship's total power consumption. An unmodified Investigator-class ship requires 39.6 kW of power to remain operational.
  
  ** - Investigator-class research ships have higher power demand than standard P.A.C.M.A.N. generator unit can provide at optimal settings, however the ship comes equipped with additional P.A.C.M.A.N. generator unit and 67 solar panels that can provide up to 33.5 kW of power under ideal circumstances. Assuming that the solar array will provide a third of its maximum output power, recommended target power output for each P.A.C.M.A.N. generator unit is 15 kW.
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
Changes seem sane, ship looks good.  Requested a small guidebook rewrite, otherwise, happy to approve.

File: Resources/ServerInfo/_NF/Guidebook/Shipyard/Investigator.xml (hunk 1 above)
Comment (ErhardSteinhauer) [reply Resources/ServerInfo/_NF/Guidebook/Shipyard/Investigator.xml]: Good stuff

**Comment by Dysh3s** [comment]
No bar = no ATM. ATMs are placed at ships with bar\kitchens, iirc. Or so i heard.

**Comment by ErhardSteinhauer** [comment]
> No bar = no ATM. ATMs are placed at ships with bar\kitchens, iirc. Or so i heard.

Not necessarily. In any case, the ATM is there to speed up the transactions between scientist and someone buying tech from them. (In rare case a scientist even cares to read the chat and sell something at all).

**Comment by Dysh3s** [comment]
> > No bar = no ATM. ATMs are placed at ships with bar\kitchens, iirc. Or so i heard.
> 
> Not necessarily. In any case, the ATM is there to speed up the transactions between scientist and someone buying tech from them. (In rare case a scientist even cares to read the chat and sell something at all).

Yeah, that's exactly why i asked for ATM for all RND ships at some point and was answered that only ships with bars\kitchens had ATMs.

**Comment by Tych0theSynth** [comment]
> > > No bar = no ATM. ATMs are placed at ships with bar\kitchens, iirc. Or so i heard.
> > 
> > 
> > Not necessarily. In any case, the ATM is there to speed up the transactions between scientist and someone buying tech from them. (In rare case a scientist even cares to read the chat and sell something at all).
> 
> Yeah, that's exactly why i asked for ATM for all RND ships at some point and was answered that only ships with bars\kitchens had ATMs.

Not sure how long ago that was, but no questions were raised about the Lyrae having an ATM when I PR'ed that. Doubt it's an issue any more, and it makes perfect sense for sci ships to have them.

**Review by whatston3 — APPROVED** [review]

**Pull request merged**